src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -195,6 +209,12 @@ public PersonListPanel getPersonListPanel() {
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
         try {
             CommandResult commandResult = logic.execute(commandText);
+
+            if (commandText.startsWith("theme")) {
+                updateTheme(logic.getTheme());
+                helpWindow.updateHelpWindowTheme(logic.getTheme());
+            }
+
```

Comment:
I used startsWith to avoid interacting too in-depth with other logic class (like via any command parser), not sure if this is the right way though, as it feels a bit brute force-ish.